scripts/generator/graphql_generator/csharp.rb

```diff
@@ -216,6 +216,12 @@ def response_init_interface(field)
       "Unknown#{type.classify_name}.Create((Dictionary<string,object>) dataJSON[key])"
     end
 
+    def response_init_union(field)
```

Comment:
Same implementation as `response_init_interface` but I duplicated it here to make it more clear.